### ISSUE TITLE
PaddleCV/image_classification 是否支持 openimages 数据集这种multi-label分类

### ISSUE THREAD

**Issue by linghaolu** [PaddlePaddle/models#4129]
PaddleCV/image_classification 是否支持 openimages 数据集这种multi-label分类 训练/预测?

**Comment by iclementine** [comment]
目前 paddle 的 cross_entropy 是支持 softlabel 的。比如一共 10 个类别，然后一个图片有 两个label 0, 和 3，那么soft label 可以设置为 (0.5, 0, 0, 0.5, 0, 0, 0, 0, 0, 0).

但是这样子需要对数据处理和网络中 loss 计算的部分做一些修改。你可以尝试修改一下。

**Comment by linghaolu** [comment]
现在有这样的例子么，paddle 的 github 上？

**Comment by iclementine** [comment]
完整的例子，似乎没有找到。
不过可以参考 https://www.paddlepaddle.org.cn/documentation/docs/zh/api_cn/layers_cn/cross_entropy_cn.html#cross-entropy
使用 soft_label.
